mne_connectivity/spectral/time.py

```diff
@@ -1,137 +1,275 @@
+    average : bool
+        Average connectivity scores over epochs. If True, output will be
+        an instance of ``SpectralConnectivity`` , otherwise
+        ``EpochSpectralConnectivity``. By default, False.
     indices : tuple of array | None
         Two arrays with indices of connections for which to compute
         connectivity. I.e. it is a ``(n_pairs, 2)`` array essentially.
         If None, all connections are computed.
     sfreq : float
-        The sampling frequency.
-    foi : array_like | None
-        Extract frequencies of interest. This parameters should be an array of
-        shapes (n_foi, 2) defining where each band of interest start and
-        finish.
+        The sampling frequency. Should be specified if data is not ``Epochs``.
+    fmin : float | tuple of float | None
+        The lower frequency of interest. Multiple bands are defined using
+        a tuple, e.g., (8., 20.) for two bands with 8Hz and 20Hz lower freq.
+        If None, the frequency corresponding to an epoch length of 5 cycles
+        is used.
+    fmax : float | tuple of float | None
+        The upper frequency of interest. Multiple bands are defined using
+        a tuple, e.g. (13., 30.) for two band with 13Hz and 30Hz upper freq.
+        If None, sfreq/2 is used.
+    fskip : int
+        Omit every "(fskip + 1)-th" frequency bin to decimate in frequency
+        domain.
+    faverage : bool
+        Average connectivity scores for each frequency band. If True,
+        the output freqs will be a list with arrays of the frequencies
+        that were averaged. By default, False.
     sm_times : float
-        Amount of time to consider for the temporal smoothing in seconds. By
-        default, 0.5 sec smoothing is used.
+        Amount of time to consider for the temporal smoothing in seconds.
+        If zero, no temporal smoothing is applied. By default, 0.
     sm_freqs : int
         Number of points for frequency smoothing. By default, 1 is used which
         is equivalent to no smoothing.
     sm_kernel : {'square', 'hanning'}
-        Kernel type to use. Choose either 'square' or 'hanning' (default).
-    mode : str, optional
-        Spectrum estimation mode can be either: 'multitaper', or
-        'cwt_morlet'.
+        Smoothing kernel type. Choose either 'square' or 'hanning' (default).
+    mode : str
+        Time-frequency decomposition method. Can be either: 'multitaper', or
+        'cwt_morlet'. See ``tfr_array_multitaper`` and ``tfr_array_wavelet``
+        for reference.
     mt_bandwidth : float | None
-        The bandwidth of the multitaper windowing function in Hz.
-        Only used in 'multitaper' mode.
-    freqs : array
-        Array of frequencies of interest for use in time-frequency
-        decomposition method (specified by ``mode``).
+        Multitaper time bandwidth. If None, will be set to 4.0 (3 tapers).
+        Time x (Full) Bandwidth product. The number of good tapers (low-bias)
```

Comment:
What is "Time x (Full) Bandwidth product"?

Reply:
This was copypasted from the docstring of `mne.time_frequency.tfr_array_multitaper`. It goes with the same documentation up to the function `mne.time_frequency.multitaper.dpss_windows`, where the parameter `half_nbw` is called with the argument `mt_bandwidth/2`. 

I agree that it's unclear, but I'm not an expert on the multitaper implementation so don't really know how to clarify.